apps/transport/client/javascripts/map.js

```diff
@@ -431,68 +431,44 @@ function addRealTimePTMap (id, view) {
         const gtfsRT = (format.gtfs_rt !== undefined ? format.gtfs_rt : 0)
         const siri = (format.siri !== undefined ? format.siri : 0)
         const siriLite = (format.siri_lite !== undefined ? format.siri_lite : 0)
-        const countOfficial = gtfsRT + siri + siriLite
-        const countNonStandardRT = format.non_standard_rt
+        const countRealTime = gtfsRT + siri + siriLite
 
-        if (countOfficial === undefined && countNonStandardRT === 0) {
+        if (countRealTime === undefined) {
             return null
         }
 
         let bind = `<strong>${name}</strong><br/>${type}`
-        if (countOfficial) {
-            const text = countOfficial === 1 ? 'Un jeu de données standardisé' : `${countOfficial} jeux de données standardisés`
+        if (countRealTime) {
+            const text = countRealTime === 1 ? 'Un jeu de données' : `${countRealTime} jeux de données`
             const aomId = feature.properties.id
             bind += `<br/><a href="/datasets/aom/${aomId}">${text}</a>`
         }
-
-        if (countNonStandardRT) {
-            const text = 'jeu de données non officiellement référencé'
-            bind += `<br/><a href="/real_time">${text}</a>`
-        }
         layer.bindPopup(bind)
     }
 
-    const styles = {
-        stdRT: {
-            weight: 1,
-            color: 'green',
-            fillOpacity: 0.5
-        },
-        nonStdRT: {
-            weight: 1,
-            color: 'red',
-            fillOpacity: 0.3
-        },
-        both: {
-            weight: 1,
-            color: lightGreen,
-            fillOpacity: 0.5,
-            opacity: 1
-        }
-    }
-
     const style = feature => {
         const format = feature.properties.dataset_formats
-        const hasStdRT = format.gtfs_rt !== undefined ||
+        const hasRealTime = format.gtfs_rt !== undefined ||
             format.siri !== undefined ||
             format.siri_lite !== undefined
-        const hasNonStdRT = format.non_standard_rt += 0
 
-        if (hasStdRT && !hasNonStdRT) {
-            return styles.stdRT
+        if (hasRealTime) {
+            return {
+                weight: 1,
+                color: 'green',
+                fillOpacity: 0.5
+            }
         }
-        if (hasNonStdRT && !hasStdRT) {
-            return styles.nonStdRT
+
+        return {
+            weight: 1,
+            fillOpacity: 0,
```

Comment:
Pour être cohérent avec les autres cartes et avec la légende (qui est pleine) :

```suggestion
            fillOpacity: 0.6,
```